### ISSUE TITLE
[BUG] Blog

### ISSUE THREAD

**Issue by taylordeal4** [nmelhado/league-page#312]
**Describe the bug**
Blog post from Contentful will not display.

**Your League ID**
1086848843544674304
In many cases, the bug maybe related to something not yet accommodated for in League Page, or something specific to your league. Adding your league ID allows for a faster debug and bug fix.

**To Reproduce**
1. Go to 'aleagueofourownffl.vercel.app'
2. Click on 'Blog'

**Expected behavior**
Should see blog post that was created in Contenful using the Training Wheels HTG. 

**Screenshots**
![image](https://github.com/user-attachments/assets/fd13276a-0e02-477b-9760-e115f99daedb)

**Desktop (please complete the following information):**
 - OS: [Windows]
 - Browser [Chrome]
 - Version [Version 127.0.6533.72 (Official Build) (64-bit)]

**Additional context**

**Comment by Meister7K** [comment]
@taylordeal4 look at issue #309 for a temp solution. I am still working on a permanent fix and will update there when I find it.  Also, probably good to close out this issue to avoid duplicate tasks.

**Comment by taylordeal4** [comment]
Correct me if I am wrong...but issue #309 is related to the manager page not loading. This issue is reference to the Blog posts not posting from Contentful to the website.

**Comment by taylordeal4** [comment]
Also, it would be nice if the Training Wheels doc was updated. It seems Contentful has made several updates and the instructions are a bit hard to follow.

**Comment by Meister7K** [comment]
@taylordeal4 you are correct. My apologies I was looking at your issue on #302 and accidentally thought this was the same issue.

**Comment by OrbitStorm** [comment]
Was able to get the blog working on two separate occasions but both times it was only momentary (i.e., a few hours).

First attempt involved using the Contentful integration on Vercel and the second attempt involved generating a new API key and using the new tokens for both the manually entered variables _and_ the variables added by the integration. Each time the blog would appear for a little while then cease to function.

**Comment by RyzoRobinson** [comment]
Did we ever get a solution to this?
